File: 1. Begginers/1018 Banknotes.py
p = [100, 50, 20, 10, 5, 2, 1]
n = int( input( ))
c = []
print(n)
i = 0
for i in range(7):
    c.append( int (n/p[i]))
    n -= c[i] * p[i]
    print('{} nota(s) de R$ {},00'.format(c[i], p[i]))

# Time: 0.016s

File: 1. Begginers/1018-Banknotes.cpp
#include <iostream>
using namespace std;

int x;
int n[] {100, 50, 20, 10, 5, 2, 1};

main(){

    cin >> x;

    cout<< x << endl;
    for (int i = 0; i < 7; i++){
        cout<< x/n[i] << " nota(s) de R$ " << n[i] << ",00" << endl;
        x = x%n[i];
    }

}
